### PR TITLE
Fixing BTFS issue at 'make check'

### PR DESCRIPTION
**Pull request by ErwanAliasr1**
This PR is solving the last issue I had by using the BTRFS filesystem when I ran the make check.

As a result, we don't have any more hundreds of subvolume not being cleared during build time.

### CODE REVIEW

**Comment by dachary** [comment]
<pre>
Reviewed-by: Loic Dachary &lt;ldachary@redhat.com>
</pre>

as soon as make check confirms.

**Comment by ErwanAliasr1** [comment]
@dachary sounds like its good ;)

**Comment by ErwanAliasr1** [comment]
Thx !

File: qa/workunits/ceph-helpers.sh [hunk 1]
```diff
@@ -126,13 +126,13 @@ function __teardown_btrfs() {
...
-    for btrfs_dir in $btrfs_dirs
-    do
-        btrfs_subdirs=`ls -l $btrfs_base_dir/$btrfs_dir | egrep '^d' | awk '{print $9}'`
-        for btrfs_subdir in $btrfs_subdirs
-        do
-            btrfs subvolume delete $btrfs_base_dir/$btrfs_dir/$btrfs_subdir
-        done
+    current_path=`pwd`
+    # extracting the current existing subvolumes
+    for subvolume in $(cd $btrfs_base_dir; btrfs subvolume list . -t |egrep '^[0-9]' | awk '{print $4}' |grep "$btrfs_base_dir/$btrfs_dir"); do
```
Comment (tchaikov) [inline qa/workunits/ceph-helpers.sh]: i have
```
/var/ceph/ceph/qa/workunits/ceph-helpers.sh:121: teardown:  __teardown_btrfs testdir/ceph-helpers
/var/ceph/ceph/qa/workunits/ceph-helpers.sh:127: __teardown_btrfs:  local btrfs_base_dir=testdir/ceph-helpers
//var/ceph/ceph/qa/workunits/ceph-helpers.sh:129: __teardown_btrfs:  ls -l testdir/ceph-helpers
//var/ceph/ceph/qa/workunits/ceph-helpers.sh:129: __teardown_btrfs:  awk '{print $9}'
//var/ceph/ceph/qa/workunits/ceph-helpers.sh:129: __teardown_btrfs:  egrep '^d'
/var/ceph/ceph/qa/workunits/ceph-helpers.sh:129: __teardown_btrfs:  btrfs_dirs=0
//var/ceph/ceph/qa/workunits/ceph-helpers.sh:130: __teardown_btrfs:  pwd
/var/ceph/ceph/qa/workunits/ceph-helpers.sh:130: __teardown_btrfs:  current_path=/var/ceph/ceph/src
//var/ceph/ceph/qa/workunits/ceph-helpers.sh:132: __teardown_btrfs:  cd testdir/ceph-helpers
//var/ceph/ceph/qa/workunits/ceph-helpers.sh:132: __teardown_btrfs:  awk '{print $4}'
//var/ceph/ceph/qa/workunits/ceph-helpers.sh:132: __teardown_btrfs:  grep testdir/ceph-helpers/
//var/ceph/ceph/qa/workunits/ceph-helpers.sh:132: __teardown_btrfs:  egrep '^[0-9]'
//var/ceph/ceph/qa/workunits/ceph-helpers.sh:132: __teardown_btrfs:  btrfs subvolume list . -t
ERROR: can't perform the search - Operation not permitted
ERROR: can't get rootid for '.'
/var/ceph/ceph/qa/workunits/ceph-helpers.sh:123: teardown:  rm -fr testdir/ceph-helpers
rm: cannot remove 'testdir/ceph-helpers/0/current': Operation not permitted
rm: cannot remove 'testdir/ceph-helpers/0/snap_1': Operation not permitted
rm: cannot remove 'testdir/ceph-helpers/0/test_subvol': Operation not permitted
rm: cannot remove 'testdir/ceph-helpers/0/sync_snap_test': Operation not permitted
rm: cannot remove 'testdir/ceph-helpers/0/async_snap_test': Operation not permitted
/var/ceph/ceph/qa/workunits/ceph-helpers.sh:94: setup:  return 1
/var/ceph/ceph/qa/workunits/ceph-helpers.sh:623: test_activate_osd:  return 1
/var/ceph/ceph/qa/workunits/ceph-helpers.sh:1486: run_tests:  return 1
```

when running "make check". #8786 is posted to address this.

**Pull request merged**